community/connections/AnyRail/index.shtml

```diff
@@ -101,10 +103,30 @@
           </ul>
         </li>
         <li>If changes have been made to the imported file, store the changes using <strong>File &rArr;
-        Store Panels...</strong> in Layout Editor or <strong>Panels &rArr; Store Panels...</strong>
+        Store Panels...</strong> in Layout Editor or <strong>File &rArr; Store ALL table content and panels...</strong>
```

Comment:
All of the various store menu items, main PanelPro menu, panel menus and table menus,  use `Store ALL table content and panels...`.